src/main/java/mate/controller/driver/AddDriverController.java

```diff
@@ -25,7 +25,8 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp)
             throws IOException, ServletException {
         String name = req.getParameter("name");
         String licenceNumber = req.getParameter("licence_number");
-        Driver driver = new Driver(name, licenceNumber);
+        Driver driver = new Driver(name, licenceNumber, req.getParameter("login"),
+                req.getParameter("password"));
```

Comment:
```suggestion
        String login = req.getParameter("login");
        String password = req.getParameter("password");
        Driver driver = new Driver(name, licenceNumber, login, password);
```

Lets make our code more consistent